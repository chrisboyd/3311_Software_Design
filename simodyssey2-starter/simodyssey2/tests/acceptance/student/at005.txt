--test explorer death due to blackhole
play
move(E)
move(SE)
move(S)




