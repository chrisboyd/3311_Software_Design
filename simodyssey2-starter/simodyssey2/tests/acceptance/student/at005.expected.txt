  state:0.0, ok
  Welcome! Try test(3,5,7,15,30)
->play
  state:1.0, mode:play, ok
  Movement:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E---   *---   P---   ----   PP--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    MY--   ----   O---   PW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   M*--   P---   P*--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    BW--   ----   *---   ----   PW--   
->move(E)
  state:2.0, mode:play, ok
  Movement:
    [0,E]:[1,1,1]->[1,2,2]
    [3,P]:[1,5,2]->[5,4,1]
    [8,P]:[4,3,1]->[3,2,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *E--   P---   ----   P---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    MY--   P---   O---   PW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   M*--   ----   P*--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    BW--   ----   *---   P---   PW--   
->move(SE)
  state:3.0, mode:play, ok
  Movement:
    [0,E]:[1,2,2]->[2,3,2]
    [5,M]:[3,1,1]->[3,5,1]
    [8,P]:[3,2,1]->[2,1,1]
    [10,B]:[5,1,1]->[5,2,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *---   P---   ----   P---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    P---   ----   WE--   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -Y--   ----   O---   PW--   M---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   M*--   ----   P*--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -W--   B---   *---   P---   PW--   
->move(S)
  state:4.0, mode:play, ok
  Explorer got devoured by blackhole (id: -1) at Sector:3:3
  The game has ended. You can start a new game.
  Movement:
    [0,E]:[2,3,2]->[3,3,2]
    [1,P]:[1,3,1]->[2,4,1]
    [2,P]:[1,5,1]->[2,5,2]
    [4,P]:[2,5,1]->[2,4,2]
    [6,P]:[3,4,1]->[2,3,2]
    [7,M]:[4,2,1]->[5,1,1]
    [10,B]:[5,2,1]->[5,1,3]
      reproduced [12,B] at [5,1,4]
      destroyed [7,M] at [5,1,1]
    [11,P]:[5,5,1]->[4,4,3]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *---   ----   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    P---   ----   WP--   PP--   -P--   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -Y--   ----   O---   -W--   M---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   -*--   ----   P*P-   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -WBB   ----   *---   P---   -W--   
